core/src/main/java/org/frankframework/pipes/IteratingPipe.java

```diff
@@ -313,7 +312,7 @@ public StopReason handleItem(I item) throws SenderException, TimeoutException, I
 							}
 						}
 						ParallelSenderExecutor pse = new ParallelSenderExecutor(sender, message, session, senderStatistics);
-						pse.setSemaphore(childThreadSemaphore);
+						pse.setSemaphore(childLimiter);
```

Comment:
I think changing the name of this field in the `ParallelSenderExecutor` makes it clearer there too.

Reply:
I agree. Wil change it too.